README.md

```
@@ -1 +1,22 @@
+
+<br>
+
+## 📚 서비스 소개
+> 독서 기반 토론 서비스 
+
+<br>
+
+## 👥 팀원 소개
+
+### 🤖 Android
+| <img src="https://avatars.githubusercontent.com/u/82762769?v=4.png" width="100" height="100"> | <img src="https://avatars.githubusercontent.com/u/192606356?v=4.png" width="100" height="100"> | <img src="https://avatars.githubusercontent.com/u/84930748?v=4.png" width="100" height="100"> |
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)